CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing to OpenShift Components
+If you are interested in contributing to OpenShift Origin, you can view all our [put-resources-here](#) that will help you get set up and provide more information.
+
+## Contributing to docs
+- section needs to be filled 
```

Comment:
you can include the content you had earlier (or just the link to the docs contribution guide)